README.md

```diff
@@ -108,4 +113,4 @@ Extra variables not required locally
 
 ## Licence
 
-[MIT Licence (MIT)](LICENCE)
+[MIT Licence (MIT)](https://opensource.org/licenses/MIT)
```

Comment:
Question: is MIT the correct licence for this repository?

I'm not sure what the convention is within HMPPS Digital.

Reply:
@rj-adams should it actually be [Open Government Licence v3.0](https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)?

That seems to be what the Prison API and Community API are using.

Reply:
Current guidance: https://ministryofjustice.github.io/technical-guidance/documentation/standards/licencing-software-or-code.html#licensing-software-or-code

I think the steer is to have content licenced via OGL and code is licenced via MIT.